.github/workflows/pull_request.yml

```diff
@@ -161,9 +161,19 @@ jobs:
           printf $SHA > .application-version
       - name: Build
         run: >
-          docker build -t onsdigital/eq-questionnaire-runner:$TAG .
+          docker build -t onsdigital/eq-questionnaire-runner:$TAG
+          -t eu.gcr.io/${{ secrets.GCLOUD_PROJECT_ID }}/eq-questionnaire-runner:$TAG .
       - name: Push to Docker Hub
         run: |
           echo ${{ secrets.DOCKER_PASSWORD }} | docker login -u ${{ secrets.DOCKER_USERNAME }} --password-stdin
           echo "Pushing to DockerHub with tag $TAG"
           docker push onsdigital/eq-questionnaire-runner:$TAG
+      - name: Push to GCR
+        env:
+          DOCKER_BUILDKIT: 1
```

Comment:
This isn't needed and should be removed.

Reply:
Removed, thanks!